Gemfile

```diff
@@ -9,6 +9,16 @@ gem 'jquery-rails'
 
 gem 'sassc'
 
+gem 'redis'
```

Comment:
It's good to leave a comment above with a description what that's gem gem for or what gem is doing